### DISCUSSION TITLE
how to reverse sort file list by date (newest first) ?

### DISCUSSION THREAD

**Discussion by sconnet** [massivemadness/Squircle-CE#222] [Help]
Scrolling all the way to the bottom of hundreds of files is a pain. Most apps let you do a sort by date (newest vs oldest). Seems Squircle only allows oldest first. Unless there is a setting in a not so obvious place?

This is for Android version. Not sure if there are other versions or not.

**Comment by massivemadness** [comment]
This will be changed in the next version (366f31deeec92602d029fc75a62512f7ed0cdbda)
